src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -18,6 +18,24 @@ function isPreview() {
 }
 
 $(document).ready(function() {
+
+	/**
+	 * Handles Keyup and Keydown on Text question to display response length
```

Comment:
Actually I reseted my eclipse yesterday and forgot to change the indentation from tab to space. sorry for that .